src/store/RelayPendingQueryTracker.js

```diff
@@ -118,12 +118,14 @@ class PendingFetch {
   _errors: Array<?Error>;
 
   constructor(
+    /* eslint-disable comma-dangle */
```

Comment:
Yes, and these results should be with 'always-multiline' enabled. So there is something in babel-eslint that gets confused in this rare case.